### PR TITLE
[DSS-353] Panel Controls spec realignment

### PR DESCRIPTION
**Pull request by monkeypox8**
## Description
DSS-353

Addresses mismatched heights and borders in Panel Controls. Because this component has been deprecated, some styles were adapted from the Toolbar component.

Note that some hover/focus states will not be updated due to the number of sub-components and edge cases involved. A few may need adjustments on KP as well.

## Screenshots

|  |Before  |  After  |
|----|----|--------|
|Component|panel-controls-before|panel-controls-after|
|People list|contacts-before|contacts-after|

## Testing in `sage-lib`
1. Navigate to the Panel Controls preview in [Rails](http://localhost:4000/pages/component/panel_controls?tab=preview) and [React](http://localhost:4100/?path=/docs/sage-panelcontrols--default)
2. Verify that heights of Panel Controls child components match

## Testing in `kajabi-products`
1. (**MED**) https://github.com/Kajabi/sage-lib/pull/1700 Style updates in Panel Controls: affects Button and Form Input heights and borders. Cosmetic changes only, no impact on functionality expected.
   - [ ] Contacts list (Contacts -> All Contacts)
   - [ ] Affilate Users (Sales -> Affiliates -> Users)

## Related

- DSS-300

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_checkbox.scss [hunk 1]
```diff
@@ -55,16 +55,18 @@ $-checkbox-focus-outline-color: sage-color(primary, 200);
...
     .sage-panel-controls & {
       flex-flow: row;
+      height: auto;
     }
   }
 
   .sage-panel-controls__bulk-actions--checked & {
-    box-shadow: map-get($sage-toolbar-button-borders, default);
+    align-self: stretch;
+    border: 1px solid sage-color(grey, 500);
```
Comment (monkeypox8) [inline packages/sage-assets/lib/stylesheets/components/_checkbox.scss:69]: Emulates button border/box-shadow style with hybrid Bulk Actions checkbox buttons

File: packages/sage-assets/lib/stylesheets/components/_dropdown.scss [hunk 2]
```diff
@@ -78,6 +78,12 @@ $-dropdown-trigger-dropdown-offset: rem(16px) + sage-spacing(xs);
     }
   }
 
+  .sage-panel-controls__bulk-actions--checked & {
+    &:last-child {
+      margin-left: rem(-1px); /* offset border overlap of multiple buttons */
```
Comment (pixelflips) [inline packages/sage-assets/lib/stylesheets/components/_dropdown.scss:83]: 🔥

**Review by pixelflips — APPROVED** [review]
LGTM! Great work! 👍🏼 

Note: couldn't test storybook due to API, but docs and KP all look great so no concerns.

**Review by ju-Skinner — APPROVED** [review]
LGTM...Tested locally

**Review by QuintonJason — APPROVED** [review]
Good stuff 🏆

**Pull request merged**